src/hotspot/share/ci/ciInstanceKlass.cpp

```diff
@@ -770,57 +770,59 @@ void StaticFieldPrinter::do_field_helper(fieldDescriptor* fd, oop mirror, bool f
       _out->print(INT64_FORMAT, *(int64_t*)&d);
       break;
     }
+    case T_PRIMITIVE_OBJECT: // fall-through
     case T_ARRAY:  // fall-through
```

Comment:
Minor nit: do both line 773 & 774 need "fall-through" comment?

Reply:
The comment in the T_ARRAY case will remain even after the removal of Q-descriptor.
The comment in the T_PRIMITIVE_OBJECT case makes it much easier to see (with grep) that this case is simply going to go away after the removal of Q-descriptors.